src/System.Runtime.Intrinsics.Experimental/ref/System.Runtime.Intrinsics.cs

```diff
@@ -27,6 +27,32 @@ public struct Vector256<T> where T : struct
 }
 namespace System.Runtime.Intrinsics.Arm.Arm64
 {
+    public static class Aes
+    {
+        public static bool IsSupported { get { throw null; } }
+        public static Vector128<byte> Decrypt(Vector128<byte> value, Vector128<byte> roundKey) { throw null; }
+        public static Vector128<byte> Encrypt(Vector128<byte> value, Vector128<byte> roundKey) { throw null; }
+        public static Vector128<byte> MixColumns(Vector128<byte> value) { throw null; }
+        public static Vector128<byte> InverseMixColumns(Vector128<byte> value) { throw null; }
+    }
+    public static class Sha1
+    {
+        public static bool IsSupported { get { throw null; } }
+        public static Vector128<uint> HashChoose(Vector128<uint> hash_abcd, uint hash_e, Vector128<uint>wk) { throw null; }
+        public static Vector128<uint> HashMajority(Vector128<uint> hash_abcd, uint hash_e, Vector128<uint>wk) { throw null; }
+        public static Vector128<uint> HashParity(Vector128<uint> hash_abcd, uint hash_e, Vector128<uint>wk) { throw null; }
+        public static uint FixedRotate(uint hash_e) { throw null; }
+        public static Vector128<uint> SchedulePart1(Vector128<uint> w0_3, Vector128<uint> w4_7, Vector128<uint> w8_11) { throw null; }
```

Comment:
Are there better names for these parameter than `w0_3`, `w4_7`?  Are these "industry standard" names for these parameters?  If no, then I'd recommend we use better names.

Same goes for `wk`, `hash_abcd`, and `hash_e`.  These should be camelCased.

Reply:
Well, since these are the names that got into coreclr, we can go with them for now.  But can you log an issue to follow up on this @sdmaclea?  In general, we follow https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-parameters for parameter names.

Reply:
Done

Reply:
thanks!